CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Copy and pasting the git commit messages is __NOT__ enough.
 
 ## [Unreleased]
 ### Added
```

Comment:
You have made a number of other changes in this PR. 

- Fixed an unintended regression in the metrics
- Added Zoo `readthedocs` documentation
- Updated contributor instructions
- Added formatting options to the competition environment.
- Individualised the agent instances in the benchmark v0 runner.

Reply:
The PR was broken into several smaller PRs. Changelog updated to reflect the changes.